### PR TITLE
Add missing items type to NonNullableStringArray definition.

### PR DESCRIPTION
**Pull request by Hyperkid123**
The missing attribute is causing exceptions when generating JS clients.

cc @Glutexo @kruai

### CODE REVIEW

**Review by kruai — APPROVED** [review]
LGTM!

**Comment by kruai** [comment]
I'd like to get one more review on this, but I don't see this causing any problems

**Review by Glutexo — APPROVED** [review]
Having a non-nullable string array is not the smartest thing out there. But as long we have only two non-nullable components and only a handful of canonical facts, it’s bearable. Let’s not try to fix everything at once. Now, we only need the schema to be valid.

**Comment by kruai** [comment]
@Glutexo is the schema not valid? I don't see any errors...

**Comment by kruai** [comment]
After re-reading your comment (and seeing that your review is in fact an Approval), I better understand what you were trying to say. Unless someone objects shortly, I'm going to merge this.

**Review by ryandillinfelton — APPROVED** [review]

**Review by thearifismail — APPROVED** [review]
Added comment for clarity otherwise LGTM

File: swagger/api.spec.yaml [hunk 1]
```diff
@@ -758,11 +760,11 @@ components:
           bios_uuid:
             $ref: "#/components/schemas/NonNullableString"
           ip_addresses:
-            $ref: "#/components/schemas/NonNullableArray"
+            $ref: "#/components/schemas/NonNullableStringArray"
```
[the remark this answers is not in the document]
Comment (kruai) [reply swagger/api.spec.yaml:763]: What do you mean? I don't think we have any comments like that in our API spec. Also, I don't understand where you're getting the text you quoted from

**Pull request merged**